include/swift/Remote/MetadataReader.h

```diff
@@ -524,6 +610,14 @@ class MetadataReader {
       return None;
 
     return start;
+#else
+    // Since there's no Objc interop, the InstanceSize field is at offset 3
+    auto address = RemoteAddress(MetadataAddress + (sizeof(StoredSize) * 3));
```

Comment:
Should I move this somewhere else? I think it's slightly weird the method name is `readInstanceStartAndAlignmentFromClassMetadata`

Reply:
This seems like a reasonable place to look up the instance size. Let's not rename `readInstanceStartAndAlignmentFromClassMetadata` in this PR, that sounds like something to tackle in a separate/follow-up PR.